app/api/services/suppliers.py

```diff
@@ -115,3 +117,50 @@ def get_suppliers_with_rejected_price(self):
         )
 
         return [r._asdict() for r in result]
+
+    def get_suppliers_with_expiring_documents(self, days):
+        today = datetime.now(pytz.timezone('Australia/Sydney'))
```

Comment:
Pendulum probably does this under the hood anyway, but `pendulum.now('Australia/Sydney')` would be equivalent if you used Pendulum here.